loom/common/core/performance.cpp

```diff
@@ -233,14 +233,6 @@ void LoomProfiler::reset()
 
 LoomProfilerRoot::LoomProfilerRoot(const char *name)
 {
-    for (LoomProfilerRoot *walk = sRootList; walk; walk = walk->mNextRoot)
```

Comment:
But this was there to catch dupes on the C++ macros. Ie PROFILE_START(a); PROFILE_START(a); Do we still catch that elsewhere?